spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -406,6 +542,8 @@ July 27, 2020 - Re-addition of source field
 
 Nov 11, 2022 - Addition of a memo field
 
+Sep 22, 2023 - Support for multi-token packets
```

Comment:
Should we update line 11 (ICS header, modified field) accordingly?

Reply:
Sounds good. And we should also probably clear the `version compatibility` field until this is released in ibc-go.